src/main/java/uk/co/ramp/covid/simulation/population/Person.java

```diff
@@ -226,4 +248,27 @@ public Sex getSex() {
     public int getAge() {
         return age;
     }
+
+    public boolean wasTested() {
+        return testOutcome.isPresent();
+    }
+
+    public void getTested() {
+        if (cVirus == null || wasTested() || !cVirus.isSymptomatic()) {
```

Comment:
What does wasTested() achieve here. Is it sayingt hat if the person has bene tested previously then they can not be tested again. If so thenw e should drop the requirement

Reply:
It's meant to be to only let them get tested twice. If they are allowed to get tested again when should this happen?

Reply:
My bad. Well, sort of. I was thinking of people being tested when they don't have Covid, but that doesn't apply here. It is possible that people are tested multiple times during a single Covid infection, but it is a reasonable asusmption that they are infected once during infection. 
We may later add a further test for being tested once the patient enters phase 2 but this sin't critical just now